### PR TITLE
EasyList Dutch needs an update: it's very outdated

### PR DESCRIPTION
**Pull request by JohnyP36**
EasyList Dutch has many rules for domains that don't exist anymore. Also sites has changed so old rules should be deleted and new ones added.

### CODE REVIEW

File: easylistdutch/hide_whitelist.txt [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
 voetbalnieuws.be#@#a[href^="http://adserving.unibet.com/"]
-tweakers.net#@#a[href^="http://pubads.g.doubleclick.net/"]
+wereldklokken.nl#@#.adsbygoogle
+xxxplaza.nl#@##ad_right
+xxxplaza.nl#@##adcontainer
+zeelandnet.nl#@#.ad-card
+!Anti-Adblock
+directwonen.nl#@#.adsbox
+fok.nl#@#.advertentie
+forum.fok.nl#@##advertentie
```
Comment (BPower0036) [inline easylistdutch/hide_whitelist.txt:37]: De laatste is overbodig. daar je al op het hele domain #@#.advertentie toegepast hebt.

Comment (JohnyP36) [reply easylistdutch/hide_whitelist.txt:37]: > De laatste is overbodig. daar je al op het hele domain #@#.advertentie toegepast hebt.

fok.nl is .advertentie
forum.fok.nl heeft geen . maar een #

**Comment by JohnyP36** [comment]
@monzta can you take a look at this pull request and merge it? 
Or give me the authority to do so, if you don't have enough time?

**Comment by monzta** [comment]
@JohnyP36 Thanks for your contributions! As these are quite a lot of changes at once, please make sure your changes meet [ABP requirements](https://adblockplus.org/en/filter-lists-requirements). Most importantly anti-adblock shouldn't be tackled in this list (as it is illegal in some countries) and tracking filters should go into [EasyPrivacy](https://github.com/easylist/easylist) instead.

**Comment by JohnyP36** [comment]
@monzta **There where already anti-adblock rules in this list**; I only changed them. (In the Netherlands anti-adblock isn't illegal and thus such rules could be added in EL Dutch. 
As far as I can see now: every thing is oké and the list could be merged.

**Comment by JohnyP36** [comment]
> @monzta  … Most importantly anti-adblock shouldn't be tackled in this list (as it is illegal in some countries) ...

To be clear as crystal: EasyList Dutch already contained anti-adblock rules. As this list is only for Dutch domains and in the Netherlands (and Belgium) anti-adblock rules aren't illegal. 
So for Dutch domains anti-adblock isn't illegal + EL Dutch already contained (some) anti-adblock rules.

**Comment by monzta** [comment]
These are the requirements for ABP. Also old anti-adblock rules shouldn't be updated. Your PR also still contains tracking ffilters (e.g. `||smetrics.cat-care.nl^`). Please remove them and add them to EasyPrivacy in case they aren't included there yet.

**Comment by JohnyP36** [comment]
If I remove the anti-adblock rules some parts or the complete site wouldn't work at all or not properly

**Comment by JohnyP36** [comment]
@monzta I think I did all you want. @BPower0036 could check if there are no tracking filters in those lists? I think I removed them.

**Comment by monzta** [comment]
Sorry for the delay! 

I think these rules belong to EasyPrivacy:

```
||config-prod.choice.faktor.io/*/faktor.js$script,domain=consent.talpanetwork.com
||dev.visualwebsiteoptimizer.com/j.php$script,important,domain=www.nu.nl
@@||faktor.io^$3p,domain=consent.talpanetwork.com
```

ABP doesn't supprt `3p` and `important` filter options.

**Pull request merged**